consensus/src/consensus.rs

```diff
@@ -36,6 +36,14 @@ pub enum ConsensusMessage {
     Timeout(Timeout),
     TC(TC),
     SyncRequest(Digest, PublicKey),
+    SyncCheckpointRequest(PublicKey), // Request latest checkpoint info from sender
```

Comment:
Can you explain why is it enough to only get the latest checkpoint info?

Reply:
Right now fetching only the latest checkpoint works only because we do very basic verification and have some assumptions about state (missing) and committee (static)
Actual fast-sync (beyond this PoC) would require additional stuff which isnt in scope.

Reply:
I see, and completely agree that the implementation is out of scope.
But we can just discuss the concept of the solution - this is what we're most interested in actually.

What would be changed here if it was production grade solution?

E.g. let's say I'm a new validator and want to do a fast sync.
Current live epoch is **100**.
Do I fetch the latest checkpoint, verify it against the current committee & download the state for that checkpoint (block)? Why can I trust that state to be correct?

NOTE: Not asking for implementation changes, just conceptual (algorithmic) answer 🙂

Reply:
So, assuming I'm a new validator looking to fast-sync, and assuming the committee isn't static, I cannot trust the state fetched this way, as the current committee could be malicious. 
The main change would be a way to create a chain of trust from the previous epoch committee to the current committee, and be able to verify that the current committee validates what we expected from the previous epoch.
If we can confirm the current committee is valid, we can proceed. Ofc, if the committee never changes, this isn't an issue as far as I can tell.

Reply:
> The main change would be a way to create a chain of trust from the previous epoch committee to the current committee

Do you have a suggestion on how to easily do this?
(last question 🤞 )